tests/functional.rs

```diff
@@ -1732,7 +1732,7 @@ fn create_monero_syncer(
     let whost = env::var("MONERO_WALLET_HOST_2").unwrap_or("localhost".into());
     let lhost = env::var("MONERO_LWS_HOST").unwrap_or("localhost".into());
 
-    let lws_wallet = &format!("http://{}:38884", lhost);
+    let lws_wallet = &format!("http://{}:8443", lhost);
```

Comment:
I remapped this port from 8443 to 38884 intentionally to not use a port that is usually reserved for ssl web servers and to put it into a similar range as the other monero services.

Reply:
rollback that patch